kaiax/gasless/impl/api.go

```diff
@@ -47,29 +48,37 @@ func NewGaslessAPI(b *GaslessModule) *GaslessAPI {
 }
 
 // GaslessTxResult represents the result of checking if a transaction is a gasless transaction
-type GaslessTxResult struct {
-	IsGasless bool   `json:"isGasless"` // Whether the transaction is a gasless transaction
-	Reason    string `json:"reason"`    // Reason why the transaction is not a gasless transaction, empty if it is
+type GaslessTxResponse struct {
+	IsGasless bool    `json:"isGasless"` // Whether the transaction is a gasless transaction
+	Reason    *string `json:"reason"`    // Reason why the transaction is not a gasless transaction, empty if it is
+}
+
+func (GaslessAPI) IsGaslessTxResponse(err error) *GaslessTxResponse {
```

Comment:
Similar to totalsupply: https://github.com/kaiachain/kaia/blob/f58cc544aadff495081ea1c9ab4dc72d23f6d027/kaiax/supply/total_supply.go#L106

```suggestion
func ToResponse(err error) *GaslessTxResponse {
```

Reply:
Unlike totalsupply, there are two Get APIs in gasless, so I used IsGaslessTxResponse instead of ToResponse, but I see.